src/Plugin/MultiselectFilterField/BooleanField.php

```diff
@@ -59,7 +59,7 @@ public function getDefaultValuesLabel(): string {
       new Result($facet, 0, 0, 1),
     ];
     $facet->setResults($results);
-
+    $this->processFacetResults($facet);
```

Comment:
This was missing...since the boolean facet may have a processor that changes 1 to On etc